spec/views/posts/index_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe 'Post', type: :system do
+  describe 'Index page' do
+    before :each do
+      @user = User.create(name: 'Tom',
+                          bio: 'Teacher from Mexico.',
+                          photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
```

Comment:
- [ ] About 17 tests are failing from the Post index specs and the Post show specs. Please try to make them green 🟢 
![image](https://user-images.githubusercontent.com/46839548/223724676-b95af0cc-6fa1-4568-975d-b119ac526088.png)